README.md

```diff
@@ -124,3 +124,25 @@ Mitosis provides a handful request filters that can be combined at will. If we w
 ```java
 filter.prepare("test", asList("a", "b"), languageEquals("en").and(userAgentContains("googlebot").negate()));
 ```
+
+## Cookie Consent
+
+When building the experiment filter you may optionally configure a cookie consent check.
+
+If a cookie with the cookie consent cookie name does not exist then mitosis will not set any cookies.
+
+If a cookie with the cookie consent cookie name does exist but the value of that cookie does not match the regex provided, mitosis will not set any cookies.
+
+Note that user experiments rely on cookies so may only be run on users who have accepted the cookie consent. SEO experiments do not rely on cookies.
+
+If a cookie with the cookie consent cookie name does exist and the value of that cookie matches the regex provided, mitosis will operate as normal.
+
+```java
+@Bean
+public Filter experimentFilter() {
+    return ExperimentFilter.builder()
+        .cookieConsent("consentCookie", "^1$")
```

Comment:
What does the `"^1$"` mean here?

Reply:
Wait I answered my own question. It checks that the cookie consentCookie equals 1.

Reply:
It's a regex string

`^` start of line
`1` the literal character 1
`$` end of line

So the only cookie value that will match the regex is "1"

Reply:
Ah yeah, it was more about how the regex gets used. Looks good tho! readme was my only feedback as well.